packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -25,6 +25,27 @@ import styles from './style.scss';
 
 const MIN_ITEMS_FOR_SEARCH = 2;
 const REUSABLE_BLOCKS_CATEGORY = 'reusable';
+const ALLOWED_EMBED_VARIATIONS = [ 'core/embed' ];
+
+function allowedBlockFilter(
```

Comment:
I see that this helper is being used in the following components:
- `packages/block-editor/src/components/inserter/block-types-tab.native.js`
- `packages/block-editor/src/components/inserter/reusable-blocks-tab.native.js`
- `packages/block-editor/src/components/inserter/search-results.native.js`

Since all of them are part of the inserter group, I'm wondering if we could move it to a new file like `utils.native.js` and import it where is needed, instead of passing it as a prop that could led to potential re-renders, wdyt?

Reply:
We could also use this function here to retrieve the reusable blocks:
https://github.com/WordPress/gutenberg/blob/6f0d199cab74545821cb1c6f266e01d12e5a562d/packages/block-editor/src/components/inserter/menu.native.js#L94-L96

Reply:
I had thought about a util script at first but thought it would be overkill. But now that you mention the re-renders and the other reusable filter in `menu.native.js` I think it's a good idea. 

I'll make those changes.